packages/orchestration/src/examples/send-anywhere.flows.js

```diff
@@ -59,26 +80,43 @@ export const sendIt = async (
   const sharedLocalAccount = await sharedLocalAccountP;
   await localTransfer(seat, sharedLocalAccount, give);
 
-  void log(`completed transfer to localAccount`);
+  if (typeof info.cctpDestinationDomain !== 'number') {
+    // within the inter-chain; no CCTP needed
+    void log(`completed transfer to localAccount`);
 
-  try {
-    await sharedLocalAccount.transfer(
-      {
-        value: destAddr,
-        encoding: 'bech32',
-        chainId,
-      },
-      { denom, value: amt.value },
-    );
-    void log(`completed transfer to ${destAddr}`);
-  } catch (e) {
-    await withdrawToSeat(sharedLocalAccount, seat, give);
-    const errorMsg = `IBC Transfer failed ${q(e)}`;
-    void log(`ERROR: ${errorMsg}`);
-    seat.exit(errorMsg);
-    throw makeError(errorMsg);
-  }
+    try {
+      await sharedLocalAccount.transfer(
+        {
+          value: destAddr,
+          encoding: 'bech32',
+          chainId,
+        },
+        { denom, value: amt.value },
+      );
+      void log(`completed transfer to ${destAddr}`);
+    } catch (e) {
+      await withdrawToSeat(sharedLocalAccount, seat, give);
+      const errorMsg = `IBC Transfer failed ${q(e)}`;
+      void log(`ERROR: ${errorMsg}`);
+      seat.fail(errorMsg);
+      throw makeError(errorMsg);
+    }
+  } else {
+    // XXX is there a cleaner way to determine that this is USDC?
+    assets.some(a => a.brand === amt.brand && a.issuerName === 'USDC') ||
+      Fail`CCTP must use USDC`;
```

Comment:
```suggestion
    assets.some(a => a.brand === amt.brand && a.issuerName === 'USDC') ||
      Fail`Only USDC is transferable to ${chainId}`;
```

For when this contract supports more than IBC and CCTP

Reply:
thanks.